### PR TITLE
Sentinel exporter

### PR DESCRIPTION
**Pull request by shonge**
Changes proposed on the PR:
- Support sentinel exporter

### CODE REVIEW

File: api/redisfailover/v1/defaults.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package v1
 const (
 	defaultRedisNumber    = 3
 	defaultSentinelNumber = 3
+	defaultSentinelExporterImage = "leominov/redis_sentinel_exporter:latest"
```
Comment (ese) [inline api/redisfailover/v1/defaults.go]: Could you set an immutable tag as default value? so we can avoid unexpected breaking changes.

File: example/redisfailover/enable-exporter.yaml [hunk 2]
```diff
@@ -5,6 +5,9 @@ metadata:
 spec:
   sentinel:
     replicas: 3
+    exproter:
+      enable: true
```
Comment (ese) [inline example/redisfailover/enable-exporter.yaml]:
```suggestion
      enabled: true
```

File: api/redisfailover/v1/defaults.go (hunk 1 above)
Comment (shonge) [reply api/redisfailover/v1/defaults.go]: OK, change default sentinel exporter image to  leominov/redis_sentinel_exporter:1.3.0.

File: example/redisfailover/enable-exporter.yaml [hunk 3]
```diff
@@ -5,8 +5,11 @@ metadata:
 spec:
   sentinel:
     replicas: 3
+    exproter:
```
Comment (ese) [inline example/redisfailover/enable-exporter.yaml]:
```suggestion
    exporter:
```

**Comment by ese** [comment]
Thanks @shonge !

**Pull request merged**